src/Player.h

```diff
@@ -0,0 +1,57 @@
+#ifndef EIGHT_MINUTE_EMPIRE_PLAYER_H
+#define EIGHT_MINUTE_EMPIRE_PLAYER_H
+
+#include <string>
+#include <vector>
+#include "BiddingFacility.h"
+#include "Map.h"
+
+// Forward Declaration
+class BiddingFacility;
+
+class Player {
+
+private:
+    // TODO: Add Player Hand of Game Cards - Requires Part 4 to be complete
+    int* coins;
+    int* armies;
+    int* cities;
+    int* dateOfBirth;
+    std::string* name;
+    std::vector<std::string*> regions;          // TODO: Figure out why we need this
```

Comment:
```suggestion
    std::vector<std::string*> * regions;          // TODO: Figure out why we need this
```
A vector of pointers isn't itself a pointer, fix is above.

Reply:
Good catch.